research/blanks/trainer.py

```diff
@@ -37,6 +37,7 @@ class BlankTrainer:
     max_sequence_length: int
     batch_size: int
     lr_scheduler: AbstractLRScheduler
+    blanks_ids: List[int]
```

Comment:
Just a suggestion: Can we make it a tensor? I think then we could make get_is_blank to accept only tensor.
That is just my statically typed brain speaking.

Reply:
Not a big issue imo

Reply:
We decided that it's not important